### DISCUSSION TITLE
How To Remove Flatpak From SpiralLinux 12

### DISCUSSION THREAD

**Discussion by just-grooving** [SpiralLinux/SpiralLinux-project#308] [Q&A]
Disk space is at a premium for me, and I don't want anything related to Flatpak in my installation.

Is it safe for me to choose the option to completely uninstall Flatpak with Synaptic?  Will that cause the installation to be unstable, or cause problems with future updates of SpiralLinux because of dependencies that this potentially removes?

I am a Linux novice who has a relatively shallow understanding of Linux, so any help or suggestions is appreciated.

Thanks.

**Comment by geckolinux** [comment]
Hi there, yes, you can remove Flatpak without issues.

SpiralLinux comes with a few Flatpak runtimes preinstalled, so first remove them:
`sudo flatpak --system uninstall --all --delete-data`

Then you can search in Synaptic for any installed packages containing the term `flatpak` in the package name (just in the package _name_, **not** the in the description) and uninstall them.

**Comment by just-grooving** [comment]
Thank you for the fast response.
